frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx

```diff
@@ -239,6 +378,21 @@ export const OperatorHubPage = withFallback(
                   kind: referenceForModel(SubscriptionModel),
                   prop: 'subscription',
                 },
+                {
+                  kind: referenceForModel(ClusterServiceVersionModel),
+                  namespaced: true,
+                  isList: false,
```

Comment:
Should this be a list?
```suggestion
                  isList: true,
```